src/main/java/net/dries007/tfc/objects/blocks/plants/BlockPlantTFC.java

```diff
@@ -168,21 +203,71 @@ public double getGrowthRate(World world, BlockPos pos)
         else return ConfigTFC.GENERAL.plantGrowthRate;
     }
 
+    @Override
+    public boolean canPlaceBlockAt(World worldIn, BlockPos pos)
+    {
+        IBlockState soil = worldIn.getBlockState(pos.down());
+        return worldIn.getBlockState(pos).getBlock().isReplaceable(worldIn, pos) && this.canSustainBush(soil);
+    }
+
     @Override
     protected boolean canSustainBush(IBlockState state)
     {
+        if (plant.getIsClayMarking()) return BlocksTFC.isClay(state);
```

Comment:
This should really be a switch statement. 
```suggestion
        switch (plant.getPlantType()) {
```